python/dftd4/pyscf.py

```diff
@@ -112,6 +113,7 @@ def kernel(self) -> Tuple[float, np.ndarray]:
             mol.charge,
             lattice=lattice,
             periodic=periodic,
+            model=self.model.lower().replace(" ", ""),
```

Comment:
Why do we need to modify this here? Should this feature be available in all APIs instead (meaning starting from the Fortran one).

Reply:
Yes, that is better. I shifted it to the interface, where the dispersion model is selected.